### ISSUE TITLE
OnePlus 7 Pro 5G - Failes to allow a directory lisitng of Root on any 21.x builds

### ISSUE THREAD

**Issue by Joecowboy** [topjohnwu/Magisk#3690]
OnePlus 7 Pro 5G
Any release builds of 21.x
Fails to do a directory listing of root.
No boot loops
No lockups
Installs fine
Installed via TWRP and in Magisk Manager and both with same results.

Version 20.4 works fine can do a full directory listing in terminal  or root explorer.

Boot image from Sprint rom v10.0.3
[boot.zip](https://github.com/topjohnwu/Magisk/files/5815198/boot.zip)

This happens on the OnePlus 7 Pro EU 5G firmware and Sprint 5G firmware.

Keep up the good work and thank you.

**Comment by zgfg** [comment]
Try MiXplorer latest Beta (not Stable or Silver), find on XDA thread, OP posts

**Comment by Joecowboy** [comment]
I reloaded the OS once again following the steps to install Disable_Dm-Verity_ForceEncryption.zip along with TWRP and Magisk 21.2.  Boy, what a process to get it to work. Tried three different times and got the OnePlus 7 Pro 5G to boot finally.  Not sure if disabling DM-Verity fixed the issue not being able to explore the root with 21.x versions or what?  I upgraded today with the latest version of Magisk 21.3 seems to work fine.  You were right **_zgfg_**,  I tried FX Explorer Premium worked.  Es Explorer and Amazed from Fdroid  with "root enabled" would not explore root with Magisk 21.x even after reinstalling the firmware. However, terminal/shell would once again explore root and not hang.  

Keep up the awesome work!!!
